src/app/horizontal-scroll.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
+
+@Directive({
+  selector: '[appHorizontalScroll]',
+  standalone: true
+})
+export class HorizontalScrollDirective {
+  constructor(private element: ElementRef) {}
+
+  @HostListener('wheel', ['$event'])
+  public onScroll(event: WheelEvent) {
+    this.element.nativeElement.scrollLeft += event.deltaY;
+  }
+}
```

Comment:
was this not doable via CSS? `overflow:auto`?

Reply:
I wanted the element to scroll left and right using the mouse wheel. It was an extra thing that didn't end up working, I can just get rid of it?